doc/classes/EditorInterface.xml

```diff
@@ -205,6 +205,22 @@
 				Plays the main scene.
 			</description>
 		</method>
+		<method name="popup_dialog_centered">
+			<return type="void" />
+			<param index="0" name="dialog" type="Window" />
+			<param index="1" name="minsize" type="Vector2i" default="Vector2i(0, 0)" />
+			<description>
+				Attempts to parent the [param dialog] to the editor UI, and then calls [method Window.popup_centered] on it. The dialog must have no current parent, otherwise the method fails.
+			</description>
+		</method>
+		<method name="popup_dialog_centered_ratio">
+			<return type="void" />
+			<param index="0" name="dialog" type="Window" />
+			<param index="1" name="ratio" type="float" default="0.8" />
+			<description>
+				Attempts to parent the [param dialog] to the editor UI, and then calls [method Window.popup_centered_ratio] on it. The dialog must have no current parent, otherwise the method fails.
```

Comment:
Since auto unparent is exposed for general use, it's probably better to move this to the `Window` as well.

Something like:
```c++
bool Window::_try_parent_dialog(Node *p_node) {
	ERR_FAIL_COND_V_MSG(is_inside_tree(), false, "Attempting to parent and popup a dialog that already has a parent.");
	ERR_FAIL_NULL_V(p_node, false);
	/// probably should have getter for `unparent_when_invisible` checked to make sure wihdow have auto unparent enabled (or forec enable it).
	/// and check if it's exclusive as well.

	Window *w = p_node->get_last_exclusive_window();
	if (w && w != this) {
		w->add_child(this);
		return true;
	}
	return false;
}

void Window::popup_exclusive_centered(Node *p_node, const Size2i &p_minsize) {
	if (_try_parent_dialog(p_node)) {
		popup_centered(p_minsize);
	}
}

void Window::popup_exclusive_centered_ratio(Node *p_node, float p_ratio) {
	if (_try_parent_dialog(p_node)) {
		popup_centered_ratio(p_ratio);
	}
}
```